tests/ballerina-tools-integration-test/src/test/java/org/ballerinalang/test/packaging/PackagingSearchTestCase.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.ballerinalang.test.packaging;
+
+import org.ballerinalang.test.IntegrationTestCase;
+import org.ballerinalang.test.context.BallerinaTestException;
+import org.ballerinalang.test.context.Constant;
+import org.ballerinalang.test.context.LogLeecher;
+import org.ballerinalang.test.context.ServerInstance;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Testing searching packages from central.
+ */
+public class PackagingSearchTestCase extends IntegrationTestCase {
+    private String serverZipPath;
+    private ServerInstance ballerinaClient;
+
+    @BeforeClass()
+    public void setUp() throws BallerinaTestException, IOException {
+        serverZipPath = System.getProperty(Constant.SYSTEM_PROP_SERVER_ZIP);
```

Comment:
What does this prop refer to ?

Reply:
This prop refers to the  path of the zip file distribution